holoviews/operation/element.py

```diff
@@ -865,7 +866,11 @@ def _process(self, element, key=None, groupby=False):
         elif isinstance(data, (nw.DataFrame, nw.LazyFrame, nw.Series)):
             if isinstance(data, nw.Series):
                 data = data.to_frame()
-            data = data.filter(nw.all().is_finite())
+            data = data.filter(~nw.all().is_null())
+            if not (data.implementation == nw.Implementation.IBIS and NARWHALS_VERSION <= (2, 10, 0)):
```

Comment:
The fix has not yet been released, but is expected to be included in the next version.

Reply:
I decided to remove it